packages/interceptors-opentelemetry/src/workflow/index.ts

```diff
@@ -1,33 +1,32 @@
 import './runtime'; // Patch the Workflow isolate runtime for opentelemetry
 import * as otel from '@opentelemetry/api';
-import * as tracing from '@opentelemetry/sdk-trace-base';
 import {
   ActivityInput,
   Next,
   WorkflowInboundCallsInterceptor,
   WorkflowOutboundCallsInterceptor,
   WorkflowExecuteInput,
 } from '@temporalio/workflow';
-import { defaultDataConverter } from '@temporalio/common';
+import { extractSpanContextFromHeaders, insertSpanContextIntoHeaders } from '@temporalio/common';
 import { instrument, instrumentFromSpanContext } from '../instrumentation';
 import { ContextManager } from './context-manager';
 import { SpanExporter } from './span-exporter';
 import { SpanName } from './interfaces';
+import * as tracing from '@opentelemetry/sdk-trace-base';
 
 export * from './interfaces';
 
-export const TRACE_HEADER = 'Otel-Trace-Context';
+let tracer: undefined | otel.Tracer = undefined;
 
-export const tracer = otel.trace.getTracer('workflow');
+function getTracer(): otel.Tracer {
```

Comment:
What's the reason for this change?

Reply:
Easier on users. They don't have to remember to call this register function